### PR TITLE
IDL Pictograms/Design - action icons

### PR DESCRIPTION
**Pull request by DianaStanciulescu**
Edit action icons from launch to arrow right for resource cards linking to pages on the same site.

### CODE REVIEW

**Review by jnm2377 — APPROVED** [review]

File: src/pages/iconography/pictograms/design.mdx [hunk 1]
```diff
@@ -586,7 +586,8 @@ color, see the gradient sections on IBM Design Color Usage.
     <ResourceCard
       subTitle="Color: Gradients"
       aspectRatio="2:1"
-      href="/color/#gradients">
+      href="/color/#gradients"
```
Comment (vpicone) [inline src/pages/iconography/pictograms/design.mdx:589]: You don't want to include a slash before the `#` when linking to a particular spot
```suggestion
      href="/color#gradients"
```

**Review by vpicone — CHANGES_REQUESTED** [review]

File: src/pages/iconography/pictograms/usage.mdx [hunk 2]
```diff
@@ -90,7 +90,8 @@ accessibility, see the section on mini units on the 2X Grid page.
     <ResourceCard
       subTitle="2X Grid: The mini unit"
       aspectRatio="2:1"
-      href="https://www.ibm.com/design/language/2x-grid#the-mini-unit">
+      href="https://www.ibm.com/design/language/2x-grid#the-mini-unit"
```
Comment (vpicone) [inline src/pages/iconography/pictograms/usage.mdx:93]: This one is correct

**Review by vpicone — CHANGES_REQUESTED** [review]
Minor href change

**Comment by DianaStanciulescu** [comment]
thanks @vpicone! I did not check the rest of the resource, just changed the icon, great observation!

**Review by vpicone — APPROVED** [review]

**Pull request merged**